rubocop-rails.yml

```
@@ -58,6 +58,10 @@ Rails/ActiveRecordCallbacksOrder:
 Rails/FindById:
   Enabled: true
 
+# https://docs.rubocop.org/projects/rails/en/latest/cops_rails/#railsfindeach
+Rails/FindEach:
+  Enabled: false
```

```suggestion
  Enabled: true
```
Creo que deberiamos de activar esta regla. Usar find_each implica que va haciendo batching de los elementos y por tanto no necesitas meter en memoria todos los elementos de la BD a la vez.